pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -25,12 +25,24 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/vm/vmstate"
 )
 
+// Exception is a type used for VM fault messages (aka exceptions). If any of
+// unwrapper functions encounters a FAULT VM state it creates an instance of
+// this type as an error using exception string. It can be used with [errors.As]
+// to get the exact message from VM and compare with known contract-specific
+// errors.
```

Comment:
So it's not supposed to work with `errors.Is`?

Reply:
It can't, all instances are different.